### DISCUSSION TITLE
小米净水器厨上版滤芯区分不开

### DISCUSSION THREAD

**Discussion by devang2** [XiaoMi/ha_xiaomi_home#93] [Ideas]
实体名、友好名称，都无法区分是哪一个滤芯。

控制
滤芯 重置滤芯
滤芯 重置滤芯
滤芯 重置滤芯
滤芯 重置滤芯

type: entities
entities:
  - entity: button.yunmi_cn_1283268_v2_reset_filter_life_a_3_1
    name: " 滤芯 重置滤芯"
  - entity: button.yunmi_cn_1283268_v2_reset_filter_life_a_5_1
    name: " 滤芯 重置滤芯"
  - entity: button.yunmi_cn_1283268_v2_reset_filter_life_a_6_1
    name: " 滤芯 重置滤芯"
  - entity: button.yunmi_cn_1283268_v2_reset_filter_life_a_7_1
    name: " 滤芯 重置滤芯"
title: 小米净水器厨上版2

**Comment by JimmyKmi** [comment]
合并：https://github.com/XiaoMi/ha_xiaomi_home/issues/94
